pkg/cloud/services/networking/router.go

```diff
@@ -45,6 +45,11 @@ func (s *Service) ReconcileRouter(openStackCluster *infrav1.OpenStackCluster, cl
 	}
 
 	routerName := getRouterName(clusterName)
+	existingRouter := false
+	if openStackCluster.Spec.RouterName != "" {
+		routerName = openStackCluster.Spec.RouterName
+		existingRouter = true
+	}
 	s.scope.Logger.Info("Reconciling router", "name", routerName)
```

Comment:
guess this line can be put into existing = false part?

Reply:
done